packages/maskbook/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "@dimensiondev/maskbook",
   "private": true,
   "license": "AGPL-3.0-or-later",
-  "scripts": {},
+  "scripts": {
+    "lint:typecoverage": "type-coverage"
```

Comment:
Did you run this lint in CI?

Reply:
ye

Reply:
![image](https://user-images.githubusercontent.com/3842474/111291322-0d2f3480-8682-11eb-91e1-8d7f5f9d374f.png)